includes/templates/omise-wp-admin-page.php

```diff
@@ -5,17 +5,13 @@
 
 		<?php Omise_Util::render_partial( 'header', $viewData ); ?>
 
-		<h1>Transactions History</h1>
-
-		<h2 class="nav-tab-wrapper wp-clearfix">
-			<a href="<?php echo esc_url( add_query_arg( array( 'page' => 'omise-plugin-admin-page' ), admin_url( 'admin.php' ) ) ); ?>" class="nav-tab<?php if ( ! isset( $_GET['action'] ) || isset( $_GET['action'] ) && 'locations' != $_GET['action'] ) echo ' nav-tab-active'; ?>"><?php esc_html_e( 'Charges' ); ?></a>
-		</h2>
+		<h1><?php echo Omise_Util::translate( 'Transactions History' ); ?></h1>
```

Comment:
@nimid according to Omise dashboard, we use 'charges' for charges list, 'transactions' for something like 
<img width="151" alt="screen shot 2559-08-18 at 7 04 02 pm" src="https://cloud.githubusercontent.com/assets/2154669/17773075/9e41d48c-6576-11e6-993b-3c7e21b0d0e3.png">

so, if that table's gonna show `charge list` I think the table's title should be `Charges History` instead of `Transactions History`